src/components/select/select.js

```diff
@@ -678,6 +681,7 @@ function SelectMenuDirective($parse, $mdUtil, $mdConstant, $mdTheming) {
   function SelectMenuController($scope, $attrs, $element) {
     var self = this;
     self.isMultiple = angular.isDefined($attrs.multiple);
+    self.multipleSeparator = $attrs.multipleSeparator || ', ';
```

Comment:
This looks like the right place to initialize this variable. The one in the `SelectDirective` doesn't seem to be used and can be removed?